src/controllers/commentsController.js

```diff
@@ -0,0 +1,169 @@
+import commentQueries from '../models/queries/commentQueries';
+import ArticleQueries from '../models/queries/articleQueries';
+
+// eslint-disable-next-line require-jsdoc
+class Comment {
+  /**
+   * Adds two numbers together.
+   * @param { object } req the request.
+   * @param { object } res The response.
+   * @returns { object } the return object.
+   */
+  static async createComment(req, res) {
+    try {
+      const userid = 1;
+      const article = await ArticleQueries.oneArticle({
+        id: req.params.id
+      });
+      if (!article) {
+        return res.status(404).send({
+          status: 404,
+          message: 'That article does not exist'
+        });
+      }
+      const comment = await commentQueries.createComment({
+        articleId: req.params.id,
+        userId: userid,
+        body: req.body.body
+      });
+
+      return res.status(201).send({
+        status: 201,
+        message: 'Comment successfully created',
+        data: comment
+      });
+    } catch (error) {
+      return res.status(500).send({
+        status: 500,
+        message: 'Ooops, something went wrong'
+      });
+    }
+  }
+
+  /**
+   * Adds two numbers together.
+   * @param { object } req the request.
+   * @param { object } res The response.
+   * @returns { object } the return object.
+   */
+  static async getComments(req, res) {
```

Comment:
Function `getComments` has 30 lines of code (exceeds 25 allowed), consider refactoring